rust/processor/src/worker.rs

```diff
@@ -356,6 +356,16 @@ impl Worker {
                 "[Parser] Successfully fetched transaction batches from channel."
             );
 
+            let last_transaction_pb_timestamp = transactions_batches
+                .last()
+                .unwrap()
+                .transactions
+                .as_slice()
+                .last()
+                .unwrap()
+                .timestamp
+                .clone();
+
```

Comment:
uh can't you just reuse batch_end_txn_timestamp lol.

Reply:
done